vlib/v/checker/tests/modules/deprecated_module.out

```diff
@@ -1,8 +1,8 @@
 vlib/v/checker/tests/modules/deprecated_module/main.v:2:1: notice: module `deprecated_module.www.ttt` will be deprecated after 2999-01-01, and will become an error after 2999-06-30; use xxx.yyy
-    1 | import bbb.ccc
-    2 | import www.ttt
-      | ~~~~~~~~~~~~~~
-    3 | import xxx.yyy
+    1 | import deprecated_module.bbb.ccc
+    2 | import deprecated_module.www.ttt
+      | ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    3 | import deprecated_module.xxx.yyy
     4 |
 vlib/v/checker/tests/modules/deprecated_module/main.v:12:11: error: undefined ident: `deprecated_module.www.ttt.non_existing`
```

Comment:
I think the changes in `vlib/v/checker/tests/modules/deprecated_module/` should be reverted.
They are probably done by `v fmt -w .`, but the problem is that vfmt currently formats the files there wrongly, due to the module lookup bug, which I am still solving.
The file `vlib/v/checker/tests/modules/deprecated_module/main.v` is deliberately skipped in `cmd/tools/vtest-cleancode.v` for now.

Reply:
I already did remove it. Should I revert?